purchase_order_line_sequence/models/account_move_line.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2023 Forgeflow S.L.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class AccountMoveLine(models.Model):
+    _inherit = "account.move.line"
+
+    related_po_sequence = fields.Char(
+        string="Sequence Related PO",
+        store="True",
+        compute="_compute_related_po_sequence",
+    )
+
+    @api.depends("purchase_line_id")
+    def _compute_related_po_sequence(self):
+        for rec in self:
+            if not rec.purchase_line_id or rec.purchase_line_id.display_type:
+                rec.related_po_sequence = False
+                continue
+            if len(rec.move_id.invoice_origin.split(", ")) > 1:
```

Comment:
The origin is an string and it is not the most reliable way to get related POs. Better use something like `rec.move_id.mapped("line_ids.purchase_order_id")`